app/controllers/users_controller.rb

```diff
@@ -50,6 +53,19 @@ def destroy
     redirect_to users_path
   end
 
+  def following
+    @title = "Following"
+    @user = User.find(params[:id])
+    @users = @user.following.page params[:page]
+    render "show_follow"
+  end
+
+  def followers
+    @title = "Followers"
+    @user = User.find(params[:id])
+    @users = @user.followers.page params[:page]
+    render "show_follow"
+  end
```

Comment:
Why don't you create follows_controller.rb, then put these in there?
```suggestion
  def following
    @title = "Following"
    @users = @user.following.page params[:page]
    render "show_follow"
  end

  def followers
    @title = "Followers"
    @users = @user.followers.page params[:page]
    render "show_follow"
  end
  
  private
  
  def find_user
     @user = User.find id: params[:id]
     return if @user.present?
     
     redirect...
  end
```